src/promise-katas-1.js

```diff
@@ -30,86 +30,143 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 
 // 1 Create a function that returns a resolving promise
 // Example:
-// when we call returnPromise like 'returnPromise()' we expect our function to use the 
+// when we call returnPromise like 'returnPromise()' we expect our function to use the
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => {
+  return new Promise((resolve, reject) => {
+    resolve();
+  });
+};
 
 // 2 Create a function that returns a promise which returns 10 if it resolves succesfully
 // Example:
 // returnTen() resolves to the value 10
 
-const returnTen = () => { };
+const returnTen = () => {
+  return new Promise((resolve, reject) => {
+    resolve(10);
+  });
+};
 
 // 3 Create a function that returns a promise which returns 'string' if it resolves succesfully
 // Example:
 // returnString() resolves to the value 'string'
 
-const returnString = () => { };
+const returnString = () => {
+  return new Promise((resolve, reject) => {
+    resolve("string");
+  });
+};
 
 // 4 Create a function that returns a promise which returns an object with a key of name and value of 'Bob' if it resolves succesfully
 // Example:
 // returnBob() resolves to the value { name: 'Bob' }
 
-const returnBob = () => { };
+const returnBob = () => {
+  return new Promise((resolve, reject) => {
+    const person = { name: "Bob" };
+    resolve(person);
```

Comment:
Dont need to create a variable for something that only gets used once, can just return it directly:
```
    resolve({ name: "Bob" });
```